### ISSUE TITLE
CG5 Week song issues.

### ISSUE THREAD

**Issue by VocalFan** [gusborg88/fnf-porter#16]
### Issue Checklist

- [X] I have properly named the issue
- [X] I looked in issues/discussions, if my question was previously anwsered

### What is the preset selected?

Full mod

### Additional Context (if applicable)

![image](https://github.com/gusborg88/fnf-porter/assets/45863583/873a3b49-b694-43d2-bdb6-a706ea3f83f0)

Splash notes are frozen, background is offset, Oreo-GF is constantly using her train hair anim instead of her head bop.

**Comment by VocalFan** [comment]
https://github.com/gusborg88/fnf-porter/assets/45863583/5fa3d7ab-170c-4f05-af2f-484483dc4865

**Comment by VocalFan** [comment]
![image](https://github.com/gusborg88/fnf-porter/assets/45863583/e5047cc9-667c-4b7f-9610-0bc4d75dd790)
Issues with GF and Splash seem consistent between mods, along with the background offset.

**Comment by BombasticTom** [comment]
![Screenshot_20240523_093130_edit_464293093252416](https://github.com/gusborg88/fnf-porter/assets/85838311/0d776670-d19a-4b3c-a46e-c0bbc6ec2700)

**Comment by BombasticTom** [comment]
It's a dev only server

**Comment by gusborg88** [comment]
yeah, we'll probably need to skip note splashes since i dont think base game supports them. And maybe a stage moving prompt.

**Comment by BombasticTom** [comment]
> yeah, we'll probably need to skip note splashes since i dont think base game supports them. And maybe a stage moving prompt.

silly! you forgot we had XML conversion tools

**Comment by MemTheBun** [comment]
1.just dont use splashes dude or else get this, https://gamebanana.com/mods/514388
2. open stage editor zoom out shift click offset object you get x and y num, got to stage jsons and change it, if characters overlap stage parts play with the z axis
3. use gf json as an example of left and right dance behaviour using specific xml frames
4. as said, classic game behaviour

**Comment by VocalFan** [comment]
@MemTheBun I actually decided to become one of the main devs and: Note splashes are gonna be properly converted later, I fixed stage offsets, character positions need properly converted, and I fixed the GF head bop.
